### ISSUE TITLE
Font awesome errors with phenogrid 1.2.3

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/monarch-legacy#1130]
We're getting font awesome when loading the phenogrid:

```
    "NetworkError: 502 Proxy Error - http://beta.monarchinitiative.org/fonts/fontawesome-webfont.woff2?v=4.5.0"

    "NetworkError: 500 Internal Server Error - http://beta.monarchinitiative.org/fonts/fontawesome-webfont.woff?v=4.5.0"

    "NetworkError: 502 Proxy Error - http://beta.monarchinitiative.org/fonts/fontawesome-webfont.ttf?v=4.5.0"
```

@yuanzhou should monarch-app be importing font awesome?

**Comment by yuanzhou** [comment]
I noticed this error today and talked to @DoctorBud, monarch-app doesn't use font awesome. Must be the assets loading path rewriting caused this issue. We also noticed that some fonts couldn't be delivered by the nodejs server, we prob just need to add some new content-types.

**Comment by yuanzhou** [comment]
In the updated phenogrid v1.2.3, after bundling we'll have all the font files placed in "phenogrid/dist/fonts" folder and they are only used by phenogrid-bundle.css. In monarch-app, probably in the webapp.js, we have some path parsing rules when loading phenogrid bundled assets. 

Similarly, there's a "phenogrid/dist/images" folder that contains all the images used in jquery-ui.css (phenogrid uses jquery-ui dialog for those FAQ popups), this path might also be rewritten by webapp.js. I'm also thinking about getting rid of the jquery-ui.css from Phenogrid and find another pop up replacement.

**Comment by yuanzhou** [comment]
This issue has been fixed in PR #1135
